packages/react/src/components/UIShell/components/styles/_side-nav.scss

```diff
@@ -110,4 +124,22 @@ div:has(.#{$prefix}--header)
 
     text-align: start;
   }
+
+  // Side Nav Link Popover
+  .#{$prefix}--icon-tooltip {
+    @include custom-property.declaration(
+      'tooltip-padding-block',
+      convert.to-rem(16px)
+    );
+  }
```

Comment:
So I actually don't know if we need this, let's confirm with @oliviaflory, I think we may just want the default tooltip spacing here. (I have seen it both ways in different design files)


@oliviaflory ignore the lack of dropshadow/theme (we are adding that upstream in core and will update here next week). Which spacing is correct.
![Screenshot 2025-02-20 at 10 13 33 AM](https://github.com/user-attachments/assets/70ac91d8-a983-41dc-9500-b09808c7bfc6)

![Screenshot 2025-02-20 at 10 13 48 AM](https://github.com/user-attachments/assets/157715ae-6743-432f-a574-338acadd30a3)

Reply:
yeah I originally had it as the top image but styled it to be like the bottom image based on #418

Reply:
hmm, I had done the custom styling to match the popover if we were using the popover with the flyout menus. But looking at Ignacios draft PR it looks like he's got more of the default spacing

<img width="219" alt="Screenshot 2025-02-20 at 6 42 02 PM" src="https://github.com/user-attachments/assets/8414dedd-6f55-4e54-b52c-becbc9f67a87" />


I suppose the custom spacing looks a little clunky with the short text of `link` or `home`. 

@emyarod is it difficult to remove the custom spacing and go with the default? The most important thing is for the tooltip and flyout to match in the end

Reply:
@oliviaflory changing the spacing won't be difficult at all! so by default spacing do you mean changing it to match the top image in @alisonjoseph's comment?